external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -2602,12 +2589,22 @@ fn parse_dot_or_index_chain(context: &mut Context) -> Result<Exp, Box<Diagnostic
                         }
                     }
                     _ => {
-                        let n = parse_identifier(context)?;
+                        let start_loc = context.tokens.start_loc();
+                        let n = match parse_identifier(context) {
+                            Ok(id) => id,
+                            Err(diag) => {
+                                done_parsing = true;
+                                context.add_diag(*diag);
+                                let end_loc = context.tokens.previous_end_loc();
+                                let loc = make_loc(context.tokens.file_hash(), start_loc, end_loc);
+                                sp(loc, Symbol::from(""))
```

Comment:
What's this symbol here?

Reply:
But we will end up with a really strange error message `Unbound field '' in 'a::m::AnotherStruct'`

Which feels like something you maybe want to suppress? Could we add a new node or some sort of error node that the IDE can latch onto?